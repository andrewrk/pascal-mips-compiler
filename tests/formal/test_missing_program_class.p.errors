ERROR: line 1: missing program class
ERROR: line 1: class "missingProgramClass" is not defined
ERROR: line 1: class "missingProgramClass" not declared
