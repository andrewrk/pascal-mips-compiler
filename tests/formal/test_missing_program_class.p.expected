ERROR: line 1: missing program class
